package.json

```diff
@@ -30,14 +29,15 @@
   "scripts": {
     "start": "react-scripts start",
     "build": "react-scripts build",
-    "test": "react-scripts test",
```

Comment:
I'm curious why you needed to do this.

Reply:
Didn't really like how `react-scripts test` ran. IMO Jest was much cleaner as it just runs the tests and that's that.